src/components/card/card.ts

```diff
@@ -0,0 +1,48 @@
+import {Component, View, ViewEncapsulation} from 'angular2/core';
+import {CONST_EXPR, Type} from 'angular2/src/facade/lang';
+
+@Component({
+  selector: 'md-card',
+  template: `
+    <div class="md-card">
+      <ng-content></ng-content>
+    </div>
+  `,
+  styleUrls: ['./components/card/card.css'],
+  encapsulation: ViewEncapsulation.None
```

Comment:
Let's discuss this issue offline, but for consistency I say we should _always_ use view encapsulation.